src/components/icon/list.tsx

```diff
@@ -1033,6 +1034,20 @@ export function getIcon(
           />
         </svg>
       );
+    case IconName.menu:
+      return (
+        <svg
+          width="5"
+          height="21"
```

Comment:
should it be 5 and 21? It seems to be quite a bit different from other stuff.
@giardiv

Reply:
yes it should be 24px height and 24px width, you might have exported the content of the component and not the whole component
<img width="364" alt="CleanShot 2022-01-24 at 13 08 39@2x" src="https://user-images.githubusercontent.com/8116383/150780404-8bee18f9-c724-41d0-8fd3-315680063f03.png">
<img width="386" alt="CleanShot 2022-01-24 at 13 08 33@2x" src="https://user-images.githubusercontent.com/8116383/150780408-adbdd4d3-6452-4e2e-bd75-705f9e6411d4.png">

Reply:
Honestly I just copied this one: https://github.com/logicalclocks/quartz/blob/dev/src/components/header/menu-button.tsx - but I can fix the sizes